src/GraphQL/Types/Schema.cs

```diff
@@ -66,11 +75,11 @@ public void Initialize()
 
         public IObjectGraphType Subscription { get; set; }
 
-        public IServiceProvider Services { get; set; }
-
         /// <summary>
-        /// Provides the ability to filter the schema upon introspection to hide types.
+        /// The service provider used to create objects during initialization of the schema
```

Comment:
```suggestion
        /// The service provider used to create objects, such as graph types, requested by the schema.
        /// Note that most objects are created once, during schema initialization.
```
How about that? I'm just trying to help point out that graph types are (mostly) not created on an as-needed basis, like would be done for Asp Core.Net, but rather are typically created once, during schema initialization.

Reply:
I mentioned also graph types as that is what most users will relate to easiest, rather than directives, middleware, and other "back-end stuff" they may not know or care about.

Reply:
> but rather are typically created once, during schema initialization

If the schema (and types) was configured as scoped, then they will be created for each HTTP request. So wording "once" may confuse.

> most users will relate to easiest, rather than directives, middleware, and other "back-end stuff" they may not know or care about.

Users certainly may not work with these concepts, but nevertheless I am sure that these concepts should be mentioned in the comments to give the reader a good picture.

Reply:
I updated the suggestion above; please review.  If you'd like to suggest a change, feel free to do so.